### DISCUSSION TITLE
build.TIMESTAMP-folders create problems in Jenkins pipeline

### DISCUSSION THREAD

**Discussion by MarcBauer-LM** [IBM/dbb-zappbuild#372] [General]
I don't like that in the full and impact build an additional folder is created with the build label for the outputs:

https://github.com/IBM/dbb-zappbuild/blob/main/build.groovy#L471

In the Jenkinsfile, we always need to "search" for BuildList.json for further processing.
From my point of view the output directory should be used directly so that it can be used unchanged as input for the following steps (e.g. PackageBuildOutputs.groovy).

What do you think?

**Comment by dennis-behm** [comment]
Hi @MarcBauer-LM, 

A configuration parameter `createBuildOutputSubfolder` is already available to control this behaviour, if the subfolder should be created or not.

Please find the parm documented in [build-conf/README.md](https://github.com/IBM/dbb-zappbuild/blob/main/build-conf/README.md#buildproperties)

Cheers
Dennis

**Reply by MarcBauer-LM** [reply]
I'm obviously blind. Thank you very much
